citations-tool/tool/src/webapp/js/citationscript.js

```diff
@@ -671,11 +671,34 @@ function checkRequiredFields( alertMsg ) {
     return false;
   }
   
+  //If electronic citation check if at least one url is supplied
+  if($('#type_selector').val() == 'electronic'){
+      var showUrlAlert = $("input:text[name^='url_']" ).filter(function(){
```

Comment:
Up to you - it's probably slightly easier to read with a class, but there's not too much difference otherwise.